style.css

```diff
@@ -635,300 +622,230 @@ input {
   }
 
-  .main__video {
+  .video {
     background: none;
     margin-bottom: 125px;
   }
 
-  .main__video h3 {
+  .video__title {
     color: white;
     text-align: center;
     white-space: nowrap;
     word-break: keep-all;
   }
 
-  .main__video_img {
+  .video__img {
     margin: 0;
     display: inline-block;
     width: 100%;
   }
 
-  .main__video_about {
+  .video__about {
     position: relative;
     margin-top: 0;
     margin-right: 0;
     width: 100%;
   }
 
-  .main__video_about p {
+  .video__text {
     margin-right: 0;
     font-size: 20px;
     line-height: 25px;
     text-align: center;
   }
 
-  .main__video_about a {
+  .video__link {
     display: none;
     width: 0;
   }
-  
-  .main__stories {
+
+  .stories {
     margin-bottom: 125px;
   }
-  .main__stories-title, .main__inst-title, .main__stories_title, .main__stories_text, .main__stories_learn-more, .main__stories_date{
+
+  .stories-title
+  ,
+  .inst-title
+  ,
+  .stories__text
+  ,
+  .stories__learn-more
+  ,
+  .stories__date {
     color: white;
   }
 
-  .main__stories_item:nth-child(n+3) {
+  .stories__content:nth-child(n+3) {
     display: none;
     max-width: none;
-  } 
-  
-  .main__stories_item {
+  }
+
+  .stories__content {
     flex-basis: 320px;
   }
 
-  .main__stories_title {
+  .stories__title {
     font-size: 24px;
     line-height: 23px;
     letter-spacing: 0;
   }
 
-  .main__stories_text {
+  .stories__text {
     font-size: 20px;
     line-height: 25px;
     margin-block-start: 0.5em;
     margin-block-end: 2.5em;
   }
 
-  .main__stories_learn-more {
+  .stories__link {
     font-size: 20px;
   }
 
-  .main__stories_date {
+  .stories__date {
     font-size: 15px;
   }
-  
-  .main__inst-title {
+
+  .inst-title {
     font-size: 30px;
     margin-block-end: 0.5em;
-    
   }
 
-  .main__inst {
+  .inst {
     flex-wrap: wrap;
     margin-bottom: 120px;
   }
 
-  .main__inst img {
+  .inst__img {
     max-width: 100%;
     height: 170px;
     flex-basis: 170px;
   }
-  
+
   .footer__wrap {
     width: 680px;
   }
 } 
 
-
-@media (max-width: 375px) { /*Стили этого запроса будут применяться только для экранных устройств с шириной области просмотра не более 375px*/
+@media (min-width: 320px) and (max-width: 375px) {
   .header {
     height: 72px;
   }
-
-  .header-left, .header-right {
+  .header_left
+  ,
+  .header_right {
     display: none;
     max-width: none;
     padding: 0;
   }
-  
-  .header__mobile-menu {
+
+  .mobile-menu {
     display: block;
     width: 100%;
   }
 
-  .header__mobile-menu img {
+  .mobile-menu__img {
     height: 22px;
     margin-left: 20px;
   }
 
-  .header__mobile-menu_icon span {
+  .mobile-menu__icon span {
     top: 5px;
-    right: 0; 
+    right: 0;
     height: 5px;
     width: 40px;
   }
 
-  .header__mobile-menu_icon span:after {
+  .mobile-menu__icon span:after {
     top: 10px;
-    right: 0; 
+    right: 0;
     height: 5px;
     width: 40px;
-    
   }
-
-  .main__shop {
+  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Mandri_Firsttask/pullRequest?prid=5484013)